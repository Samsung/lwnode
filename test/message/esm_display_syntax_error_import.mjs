/* eslint-disable no-unused-vars */
import '../common/index.mjs';
import {
  foo,
  notfound
} from '../fixtures/es-module-loaders/module-named-exports.mjs';
