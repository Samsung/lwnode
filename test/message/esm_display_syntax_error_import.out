file:///*/test/message/esm_display_syntax_error_import.mjs:5
  notfound
  ^^^^^^^^
SyntaxError: The requested module '../fixtures/es-module-loaders/module-named-exports.mjs' does not provide an export named 'notfound'
    at ModuleJob._instantiate (internal/modules/esm/module_job.js:*:*)
    at async ModuleJob.run (internal/modules/esm/module_job.js:*:*)
    at async Loader.import (internal/modules/esm/loader.js:*:*)
    at async Object.loadESM (internal/process/esm_loader.js:*:*)
